spring-test/src/main/java/org/springframework/test/context/jdbc/SqlScriptsTestExecutionListener.java

```diff
@@ -126,20 +127,19 @@ public void afterTestMethod(TestContext testContext) throws Exception {
 	 * {@link TestContext} and {@link ExecutionPhase}.
 	 */
 	private void executeSqlScripts(TestContext testContext, ExecutionPhase executionPhase) throws Exception {
-		boolean classLevel = false;
+		executeSqlScriptsByElement(testContext, executionPhase, testContext.getTestClass());
+		executeSqlScriptsByElement(testContext, executionPhase, testContext.getTestMethod());
```

Comment:
This new feature/behavior must not be enabled by default.

See [my comments in JIRA](https://jira.spring.io/browse/SPR-16021?focusedCommentId=159901&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-159901) for details.